tsl/src/continuous_aggs/refresh.c

```diff
@@ -885,11 +903,11 @@ continuous_agg_refresh_internal(const ContinuousAgg *cagg,
 	 * case.
 	 *
 	 * For variable width buckets we use a refresh_window.start value that is lower than the
-	 * -infinity value (ts_time_get_nobegin < ts_time_get_min). Therefore, the first check in the
-	 * following if statement is not enough. If the invalidation_threshold returns the min_value for
-	 * the data type, we end up with [nobegin, min_value] which is an invalid time interval.
-	 * Therefore, we have also to check if the invalidation_threshold is defined. If not, no refresh
-	 * is needed.  */
+	 * -infinity value (ts_time_get_nobegin < ts_time_get_min). Therefore, the first check in
+	 * the following if statement is not enough. If the invalidation_threshold returns the
+	 * min_value for the data type, we end up with [nobegin, min_value] which is an invalid time
+	 * interval. Therefore, we have also to check if the invalidation_threshold is defined. If
+	 * not, no refresh is needed.  */
```

Comment:
I think I've noticed it in other your PRs but why are u changing the comments format?? Something different with your clang-format?

Reply:
Not sure why. I have automatic clang-format on save, but it should pick up the config that is in the repository. I probably have a later version, which might explain why if they have different defaults.